src/status_im/subs/root.cljs

```diff
@@ -109,7 +109,8 @@
 (reg-root-key-sub :chat/memberships :chat/memberships)
 (reg-root-key-sub :group-chat/invitations :group-chat/invitations)
 (reg-root-key-sub :chats/mention-suggestions :chats/mention-suggestions)
-(reg-root-key-sub :chat/inputs-with-mentions :chat/inputs-with-mentions)
+\
```

Comment:
Just did a quick check on the PR, this seems to be a leftover @flexsurfer.

Reply:
oops! thank you

Reply:
wondering why it compiles without errors